poetry.lock

```diff
@@ -2716,7 +2716,7 @@ description = "MessagePack (de)serializer."
 name = "msgpack"
 optional = false
 python-versions = "*"
-version = "0.6.2"
+version = "1.0.0"
```

Comment:
maybe we should pin it ? see https://github.com/msgpack/msgpack-python#major-breaking-changes-in-msgpack-10

Reply:
👍